### ISSUE TITLE
Feature: color the interface with shulker color

### ISSUE THREAD

**Issue by Meeni-mc** [kyrptonaught/quickshulker#13]
When a shulker box is on the floor, opening it, opens the shulker inventory interface, and the interface is colored with the shulker color.

When opening a shulker box in inventory (right click in inventory), the shulker inventory interface is the color of the last ground-opened shulker (for example, if I open a red placed shulker, opening any color box in-inventory will use the red variant of the interface), 

Desired behavior: opening a shulker in inventory should open the shulker box interface of the same color as the shulker dye.

**Comment by kyrptonaught** [comment]
Ah I never even noticed this was missing. I'll look into a fix

**Comment by kyrptonaught** [comment]
Ah seems I was a but confused, vanilla shulker boxes do not do this by default. I thought it was a vanilla feature that was missing

**Comment by Meeni-mc** [comment]
You are right, this is part of Optifine(optifabric). Your mod does disable this feature though.

**Comment by supersaiyansubtlety** [comment]
You can also get the colored shulker backgounds through Tweakeroo or ShulkerBoxTooltip. 
I recommend trying those instead as optifine is notoriously difficult to be compatible with.

**Comment by Meeni-mc** [comment]
Tweakeroo color option applies only to the in-inventory mouse hovering peek option. This works as intended with this mod. 

Tweakeroo does not support coloring the 'on the floor' shulker. This is the part that does not work as intended with OF (possibly because it uses CIT textures to do it).

**Comment by Meeni-mc** [comment]
Just verified it's the same story with ShulkerBoxTooltip.

**Comment by MeeniMc** [comment]
I am not using OF anymore so I do not know if this issue still exists.
